mainnet/lighthouse/lighthouse.toml

```diff
@@ -14,4 +14,4 @@ enable-private-discovery = true
 enr-address = "0.0.0.0"
 metrics = true
 metrics-port = 5054
-boot-nodes = 'enr:-MK4QHdcy0ljlcSOPBGd2ydCh0-Rd-rRKqExylHiKunbJC3eI3DKSh4fGNM4pDnNYNvRu-9llVB1pMXloywn4XzVWhWGAYgmkP7yh2F0dG5ldHOIAAAAAAAAAACEZXRoMpDcUbERQgAABKQfAAAAAAAAgmlkgnY0gmlwhCKTScGJc2VjcDI1NmsxoQJNpNUERqKhA8eDDC4tovG3a59NXVOW16JDFAWXoFFTEYhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A'
\ No newline at end of file
+boot-nodes = "enr:-MK4QJ-Bt9HATy4GQawPbDDTArtnt_phuWiVVoWKhS7-DSNjVzmGKBI9xKzpyRtpeCWd3qA9737FTdkKGDgtHfF4N-6GAYlzJCVRh2F0dG5ldHOIAAAAAAAAAACEZXRoMpA2ulfbQgAABP__________gmlkgnY0gmlwhCKTScGJc2VjcDI1NmsxoQJNpNUERqKhA8eDDC4tovG3a59NXVOW16JDFAWXoFFTEYhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A,enr:-MK4QDOs4pISOkkYbVHnGYHC5EhYCsVzwguun6sFZjLTqrY6Kx_AoE-YyHvqBIHDUwyQqESC4-B3o6DigPQNfKpdhXiGAYgmPWCdh2F0dG5ldHOIAAAAAAAAAACEZXRoMpA2ulfbQgAABP__________gmlkgnY0gmlwhCIgwNOJc2VjcDI1NmsxoQNGVC8JPcsqsZPoohLP1ujAYpBfS0dBwiz4LeoUQ-k5OohzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A"
```

Comment:
It decodes as:

<img width="1359" alt="image" src="https://github.com/lukso-network/network-configs/assets/477945/74f0f8af-7b73-4989-a8a8-754ea3c7ce4a">

And

<img width="1362" alt="image" src="https://github.com/lukso-network/network-configs/assets/477945/441e8b90-6cdb-4548-9323-f1f4fc756f0f">

Reply:
no no im just putting this to help reviewing and giving more details.